src/main/java/org/lesson4/ex3/BaseEmployee.java

```diff
@@ -0,0 +1,22 @@
+package org.lesson4.ex3;
+
+public class BaseEmployee {
+    public boolean introduce;
+    String name;
+    String surname;
+    int yearOfEmployment;
+    int bonus;
+    private final int baseSalary;
+    public BaseEmployee(int baseSalary){
```

Comment:
formatowanie kodu do poprawy, bardzo to nieczytelne. gdzie odstępy między metodami?